src/styles/BorrowingCalculatorElements.tsx

```diff
@@ -153,6 +167,14 @@ export const BoxOne = styled.div`
                 font-size: 13px;
                 line-height: 15px;
                 color: #575757;
+
+                margin-left: 7px;
+                appearance: none;
+                -moz-appearance: none;
+                -webkit-appearance: none;
+                background-image: url("https://i.imgur.com/5v17RRC.png");
```

Comment:
With these `background-image:` you can actually use `background-image: ${MyImage};` and `import MyImage from '../assets/myimage.jpg';`

Reply:
I tried that!!!!! But it didn't work :( so I resorted to uploading it to imgur haha!

Reply:
Acutally I don't think I used the "$" sign... I just did `{MyImage}`